bindata/assets/alerts/api-usage.yaml

```
@@ -34,7 +34,7 @@ spec:
               a successful upgrade to the next EUS cluster version with Kubernetes {{ $labels.removed_release }}.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: >-
-            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1.3[45]"})
+            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1.3[56]"})
```

```suggestion
            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1.35"})
```
The next EUS release 4.22 will be on Kubernetes 1.35

---

Amended.